### ISSUE TITLE
IE11 issue: Permission must be Installable.

### ISSUE THREAD

**Issue by d4rky-pl** [Templarian/MaterialDesign-Webfont#60]
When trying to use the TTF font in Internet Explorer 11, it throws an error:

```
@font-face failed OpenType embedding permission check. Permission must be Installable.
```

Setting the permission on the TTF font fixes the issue. This is necessary when using libraries like `@expo/vector-icons` that use a single file for all platforms

**Comment by Templarian** [comment]
Not sure how to fix this or what this impacts exactly? Our entire build package is open source and uses the `webfont` package. 😕 

https://github.com/Templarian/MaterialDesign-Font-Build

**Comment by d4rky-pl** [comment]
Font Awesome had the same issue, this is how they solved it:
https://github.com/FortAwesome/Font-Awesome/issues/2517

tl;dr running https://github.com/hisdeedsaredust/ttembed during the build process

**Comment by d4rky-pl** [comment]
[ttembed-js](https://github.com/thegregorator/ttembed-js) is written in pure JS and solves the issue perfectly

**Comment by Templarian** [comment]
Oops, totally forgot to add this to the build process step before running `4.9.95` release.

We'll be adding this to the 5.0 release. Sorry about that.

**Comment by d4rky-pl** [comment]
@Templarian 

```
yarn ttembed-js -n materialdesignicons-webfont.ttf
fsType is 0000; no action will be taken if run without -n.
```

This issue can be closed, thank you!
